modules/prediction/prediction_component.cc

```diff
@@ -107,7 +110,7 @@ bool PredictionComponent::Proc(
 
 bool PredictionComponent::ContainerSubmoduleProcess(
     const std::shared_ptr<PerceptionObstacles>& perception_obstacles) {
-  const double frame_start_time = Clock::NowInSeconds();
+  const double frame_start_time = apollo::cyber::Time::Now().ToNanosecond();
```

Comment:
1. ToNanosecond() returns uint64.
2. Try `absl::ToUnixNanos(absl::Now())` if you need a system time.